### PR TITLE
Feature/109-Add-documentation-on-server-and-client

### LINKED ISSUE #109: Add documentation on server and client
This page should include the functionalities supported, e.g., population and exploration, with examples.

### PR DESCRIPTION
**Pull request by NoB0**
Fixes #109

### CODE REVIEW

File: docs/source/pkg_client.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Node.js and npm are required to run PKG Client, more information on how to install them can be found `here <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`.
+
+To start PKG Client, run the following commands in the directory `pkg_client`:
+
+.. code-block:: bash
+
+    npm install
+    npm start
+
+Note that `npm install` only needs to be run once, unless the dependencies change. For more information on how to run the client, refer to the README in the `pkg_client` directory.
```
Comment (WerLaj) [inline docs/source/pkg_client.rst:18]: Can we link here this README file?

File: docs/source/pkg_client.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+In case the server is not running on the default port (i.e., 5000), `PKG_API_BASE_URL` in `pkg_client/public/config.json` must be updated accordingly.
+
+Features
+--------
+
+PKG Client provides the following features:
```
Comment (WerLaj) [inline docs/source/pkg_client.rst:25]: Maybe: the following forms/pages?

File: docs/source/pkg_client.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+* **Login/Registration**: To access their PKG, users must be logged in. If they do not have an account, they can register.
+* **Home**: The home page provides a text input field where users can enter their management instructions (i.e., add, remove, and retrieve statements) in natural language. After being processed by the server, the outcome of the instructions is displayed. The home page also provides a button to visualize the PKG.
+* **Populate PKG**: Users can populate their PKG by adding or removing statements via a form. This feature assumes that the user is familiar with the `PKG vocabulary <https://iai-group.github.io/pkg-vocabulary/>`.
+* **Explore PKG**: Users can visualize their PKG or a part of it as an RDF graph. The visualization is generated based on the results of a SPARQL query, hence, this feature is intended for advised users.
```
Comment (WerLaj) [inline docs/source/pkg_client.rst:30]: Nit: advanced users

File: docs/source/pkg_server.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+By default, the server will run locally on port 5000. In case you want to run the server on a different port, you can specify the port using the `--port` flag.
+
+Routes
+------
+
+The server has four main routes that relate to the :doc:`features available <pkg_client>` in the PKG Client:
+
+* `/auth`: Handles the authentication of users and service providers.
+* `/nl`: Handles natural language instructions.
```
Comment (WerLaj) [inline docs/source/pkg_server.rst:27]: Nit: instructions provided by users to manage PKG

**Review by WerLaj — APPROVED** [review]
LGTM with some suggestions

File: docs/source/pkg_client.rst (hunk 2 above)
Comment (NoB0) [reply docs/source/pkg_client.rst:25]: Done.

File: docs/source/pkg_client.rst (hunk 3 above)
Comment (NoB0) [reply docs/source/pkg_client.rst:30]: Done.

File: docs/source/pkg_server.rst (hunk 4 above)
Comment (NoB0) [reply docs/source/pkg_server.rst:27]: Done.

File: docs/source/pkg_client.rst (hunk 1 above)
Comment (NoB0) [reply docs/source/pkg_client.rst:18]: Done.

**Pull request merged**